tests/test_prescan_sub.py

```diff
@@ -342,7 +340,146 @@ def test_prescan_sub():
             # plt.tight_layout()
             # plt.show()
 
+def test_bias_zeros_frame():
+    """Verify prescan_biassub does not break for a frame of all zeros 
+    (should return all zeros)."""
+    
+    ###### create simulated data
+    datadir = os.path.join(os.path.dirname(__file__), "simdata")
+    
+    for obstype in ['SCI', 'ENG']:
+        # create simulated data
+        dataset = mocks.create_prescan_files(filedir=datadir, obstype=obstype,numfiles=1)
+
+        # Overwrite data with zeros
+        dataset.all_data[:,:,:] = 0
+
+        for return_full_frame in [True, False]:
+            
+            output_dataset = prescan_biassub(dataset, return_full_frame=return_full_frame)
+
+            if not np.all(output_dataset.all_data == 0.):
+                raise Exception('Operating on all zero frame did not return all zero frame')
+
+def test_bias_hvoff():
+    """Verify that function finds bias for hvoff distribution."""
+    
+    # Set tolerance
+    tol = 1.
+    bval = 100.
+    seed = 12346
+
+    ###### create simulated data
+    datadir = os.path.join(os.path.dirname(__file__), "simdata")
+    
+    for obstype in ['SCI', 'ENG']:
+        # create simulated data
+        dataset = mocks.create_prescan_files(filedir=datadir, obstype=obstype,
+                                             numfiles=1)
+
+        # Overwrite data with normal distribution
+        rng = np.random.default_rng(seed)
+        dataset.all_data[:,:,:] = rng.normal(bval, 1,
+                                             size=dataset.all_data.shape)
+
+        for return_full_frame in [True, False]:
+            
+            output_dataset = prescan_biassub(dataset, return_full_frame=return_full_frame)
+
+            if output_dataset[0].ext_hdr['MED_BIAS'] - bval > tol:
```

Comment:
Change to np.abs() > tol. It works without this b/c the quantity turns out to be positive, but this change would make it clearer to someone seeing this test fresh that the subtraction is not negative, thus avoiding the tolerance check.